libvirt_exporter.go

```diff
@@ -26,6 +26,32 @@ import (
 	"github.com/kumina/libvirt_exporter/libvirt_schema"
 )
 
+/*
+#cgo LDFLAGS: -lvirt
+#include <libvirt/libvirt.h>
+#include <libvirt/virterror.h>
+#include <stdlib.h>
+*/
+import "C"
+
+const (
+	VIR_DOMAIN_MEMORY_STAT_SWAP_IN        = C.VIR_DOMAIN_MEMORY_STAT_SWAP_IN
```

Comment:
Any reason why you can't just use the constants provided by the libvirt go bindings?
https://godoc.org/github.com/libvirt/libvirt-go#DOMAIN_MEMORY_STAT_LAST

Reply:
I'm sorry that I'm fairly new to golang and didn't know that. I'll take it back and refine the change.